{"uuid": "ea4278f7-5c5a-452b-82b0-89e3d82ea947", "children": ["b3f3ccf2-a836-45a3-8f73-a6292e088c2f"], "befores": [{"name": "client", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 405 Client Error: Method Not Allowed for url: https://prod-auth-service.ambitionhire.ai/v1/auth_routes/recruiter-login/\n", "trace": "  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\Desktop\\Ambition hire file\\Ambition hire\\tests\\API\\test_login.py\", line 12, in client\n    api_client.login(\n  File \"C:\\Users\\Dell\\Desktop\\Ambition hire file\\Ambition hire\\api_client.py\", line 49, in login\n    response.raise_for_status()\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\requests\\models.py\", line 1021, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "start": 1738322462329, "stop": 1738322463066}], "start": 1738322462329, "stop": 1738322463252}